framework/Optimizers/GeneticAlgorithm.py

```diff
@@ -392,6 +392,8 @@ def _useRealization(self, info, rlz1):
     """
     # The whole skeleton should be here, this should be calling all classes and _private methods.
     traj = info['traj']
+    for t in self._activeTraj[1:]:
+      self._closeTrajectory(t, 'cancel', 'Currently GA is single trajectory',0)#, None
     self.incrementIteration(traj)
```

Comment:
I think I am not sure how removing all trajectories but the first one help convergence

Reply:
First of all, in GA we have only one trajectory but we are using the multi trajectory to allow the user to provide an initial population of solutions not a single initial solution. Now, if we left all the trajectories open, (one that has all population and the rest are empty) once the first trajectory is converged, it continues running because the remaining trajectories are not converged yet and hence it continues till the iteration limit is reached.